mgwr/summary.py

```diff
@@ -52,20 +57,20 @@ def summaryGLM(self):
                                                 '-' * 10, '-' * 10)
     for i in range(self.k):
         summary += "%-31s %10.3f %10.3f %10.3f %10.3f\n" % (
-            XNames[i][:30], glm_rslt.params[i], glm_rslt.bse[i],
+            XNames[i], glm_rslt.params[i], glm_rslt.bse[i],
```

Comment:
Why are you removing `[:30]`? That is not a stylistic change and there is a reason it is in the original code.